### PR TITLE
Example/reuse chrome for #765

### PR DESCRIPTION
**Pull request by bdurrani**
#765 

Sorry for all those merge commits showing up....again... 😢

### CODE REVIEW

File: docfx_project/examples/ReuseChrome.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Use `BrowserFetcherOptions` to specify the full path for where to download Chrome.
+
+```
+var browserFetcherOptions = new BrowserFetcherOptions { Path = downloadPath };
+var browserFetcher = new BrowserFetcher(browserFetcherOptions);
+await browserFetcher.DownloadAsync(BrowserFetcher.DefaultRevision);
+```
+
+Use `Puppeteer.launchAsync()` with `LaunchOptions` with the `LaunchOptions.ExecutablePath` property set to the
```
Comment (kblok) [inline docfx_project/examples/ReuseChrome.md]:
```suggestion
Use `Puppeteer.LaunchAsync()` with `LaunchOptions` with the `LaunchOptions.ExecutablePath` property set to the
```

File: docfx_project/examples/toc.yml [hunk 2]
```diff
@@ -8,4 +8,6 @@
       href: Page.EvaluateFunctionAsync.ComplexJSObjects.md
     - name: How to wait for a selector
       href: Page.WaitForSelectorAsync.Searching.md
+    - name: Download Chrome to a custom location and Using Chrome from a custom location
```
Comment (kblok) [inline docfx_project/examples/toc.yml]:
```suggestion
    - name: Download and Reuse Chrome from a custom location
```

**Comment by bdurrani** [comment]
fixed up the issues

File: samples/reuse-downloaded-chrome/Program.cs [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                Directory.CreateDirectory(downloadPath);
+            }
+
+            Console.WriteLine("Downloading Chromium");
+
+            var browserFetcherOptions = new BrowserFetcherOptions { Path = downloadPath };
+            var browserFetcher = new BrowserFetcher(browserFetcherOptions);
+            await browserFetcher.DownloadAsync(BrowserFetcher.DefaultRevision);
+
+            var executablePath = CreateExecutablePath(downloadPath);
```
Comment (kblok) [inline samples/reuse-downloaded-chrome/Program.cs]: `BrowserFetcher` has a [GetExecutablePath](https://github.com/kblok/puppeteer-sharp/blob/master/lib/PuppeteerSharp/BrowserFetcher.cs#L206) method that will do that for you.

Comment (bdurrani) [reply samples/reuse-downloaded-chrome/Program.cs]: Sorry I should have checked the API. Thanks for the tip.

**Comment by bdurrani** [comment]
@kblok  not sure why this test is failing for an example

**Comment by kblok** [comment]
@bdurrani just a hiccup :)

**Pull request merged**